### DISCUSSION TITLE
After disabling the checkbutton, the checked state will not be displayed

### DISCUSSION THREAD

**Discussion by cyberxnomad** [israel-dryer/ttkbootstrap#244] [Q&A] [answered]
After disabling the checkbutton(default style), the checked state will not be displayed
I want it to be unclickable, but not completely unusable. Like READONLY. Are there any related options?

**Comment by israel-dryer** [comment, marked as answer]
I need to create disabled state style images that reflect the state of the button.

**Reply by israel-dryer** [reply]
This was fixed with PR #266
